v/vertx-micrometer-metrics/vertx-micrometer-metrics_ubi8.4.sh

```diff
@@ -0,0 +1,107 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : vertx-micrometer-metrics
+# Version       : 3.9.7
+# Source repo   : https://github.com/vert-x3/vertx-micrometer-metrics
+# Tested on     : RHEL UBI 8.4
```

Comment:
Does this mean that the script has been validated on RHEL 8.4 as well as UBI 8.4? The scripts which work on UBI 8.4 almost certainly work on RHEL 8.4, but the reverse is not always true.

Reply:
It was tested on UBI 8.4